docs/user-guide/labels.md

```diff
@@ -121,15 +121,58 @@ _Set-based_ requirements can be mixed with _equality-based_ requirements. For ex
 
 LIST and WATCH operations may specify label selectors to filter the sets of objects returned using a query parameter. Both requirements are permitted:
 
-   * _equality-based_ requirements: `?labelSelector=key1%3Dvalue1,key2%3Dvalue2`
-   * _set-based_ requirements: `?labelSelector=key+in+%28value1%2Cvalue2%29%2Ckey2+notin+%28value3%29`
+    * _equality-based_ requirements: `?labelSelector=environment%3Dproduction,tier%3Dfrontend`
+    * _set-based_ requirements: `?labelSelector=environment+in+%28production%2Cqa%29%2Ctier+in+%28frontend%29`
 
-Kubernetes also currently supports two objects that use label selectors to keep track of their members, `service`s and `replicationcontroller`s:
+Both label selector styles can be used to list or watch resources via a REST client. For example targetting `apiserver` with `kubectl` and using _equality-based_ one may write:
 
-* `service`: A [service](services.md) is a configuration unit for the proxies that run on every worker node.  It is named and points to one or more pods.
+```shell
+$ kubectl get pods -l environment=production,tier=frontend
+```
+
+or using _set-based_ requirements:
+
+```
+$kubectl get pods -l 'environment in (production),tier in (frontend)'
+```
+
+As already mentioned _set-based_ requirements are more expressive.  For instance, they can implement the _OR_ operator:
+
+```shell
+$kubectl get pods -l 'environment in (production, qa)'
+```
+
+or restricting negative matching via _exists_ operator:
+
+```shell
+$kubectl get pods -l 'environment,environment notin (frontend)'
+```
+
+Kubernetes also supports two objects that use label selectors to keep track of their members, `service`s and `replicationcontroller`s:
```

Comment:
Per #13699, the main objective is to clarify the difference between the 2 different flavors of selectors.